x-pack/plugins/fleet/public/applications/fleet/components/fleet_server_instructions/steps/get_started.tsx

```diff
@@ -92,8 +93,24 @@ const GettingStartedStepContent: React.FunctionComponent<QuickStartCreateForm> =
       <EuiText>
         <FormattedMessage
           id="xpack.fleet.fleetServerSetup.getStartedInstructions"
-          defaultMessage="First, set the public IP or host name and port that agents will use to reach Fleet Server. It uses port {port} by default. We'll then generate a policy for you automatically."
-          values={{ port: <EuiCode>8220</EuiCode> }}
+          defaultMessage="First, set the public IP or host name and port that agents will use to reach Fleet Server. It uses port {port} by default {toolTip}. We'll then generate a policy for you automatically."
+          values={{
+            port: <EuiCode>8220</EuiCode>,
+            toolTip: (
+              <EuiIconTip
+                iconProps={{
+                  className: 'eui-alignTop',
+                }}
+                content={
+                  <FormattedMessage
+                    id="xpack.fleet.fleetServerSetup.getStartedInstructionsPortTooltips"
+                    defaultMessage="This can be changed during Fleet Server installation."
```

Comment:
```suggestion
                    defaultMessage="This can only be set during Fleet Server installation."
```
Do we want to change this to have consistent wording?